### ISSUE TITLE
Add Jetstream Create User command to Tinker

### ISSUE THREAD

**Issue by tedsecretsource** [laravel/jetstream#441]
This is a request to add a tinker command to allow me to add a user from within tinker and have it execute the create user action in the Actions folder (create a personal team, etc.). At present, that doesn't seem to be happening when doing `App\Models\User::create([etc])`.

**Comment by driesvints** [comment]
We have no plans to build this ourselves sorry

**Comment by tedsecretsource** [comment]
Thanks. I appreciate the quick feedback. Is there an existing way to disable user self-registration but allow a developer to create the users? I'm not opposed to trying to get this working myself but might need some guidance.

**Comment by joelbutcher** [comment]
@tedsecretsource It seems to me that you just want an artisan command for this, rather than something to execute in a tinker shell,

Something like `php artisan jetstream:create-user Taylor --teams` and have the command generate an email like `taylor@example.org`

**Comment by joelbutcher** [comment]
@tedsecretsource If you wanna check out my fork, I've written this command in a branch:

https://github.com/joelbutcher/jetstream/tree/create-user-command

**Comment by tedsecretsource** [comment]
Yes, this is exactly what I am looking for. Sorry for the poor description.

I've created a console command `jetstream:createuser {name} {email} {password}` (I'll eventually get the password off the command line) following the instructions [here](https://laravel.com/docs/8.x/artisan) and it's almost working (currently getting the error `Object of class App\Models\User could not be converted to int` but not sure why). 

Assuming I get it working, is this something Laravel would like me to contribute and if so, where (here I assume but seeing as how driesvints has already said this is not a feature they plan on building themselves, I'm not sure if he means they aren't interested or simply that I should do it myself).

Thanks for… just as I finished typing this I saw that you did the same thing LOL. What do you think about seeding the password with `$faker` or something more random so you don't have a password in the repo? People can then use the password reset feature to log in. Also, I'd put the email on the command line (if you haven't already and I didn't see it). Can I fork that branch and do a PR with these tweaks?

Also, is there any need to respect or use the `CreatesNewUsers` contract in this context?

Thanks in advance.

Ted

**Comment by joelbutcher** [comment]
@tedsecretsource by all means fork the branch / take the code and make your own from it. 

It's hard to say whether or not the guys at Laravel would accept it, best bet is just to make a PR for it and see what they say (they have merged branches they've discussed not implementing the content for, themselves)

**Comment by tedsecretsource** [comment]
@joelbutcher I'm in over my head on this. I created a command that does almost exactly what yours does but with some tweaks to suit my needs. I was thinking about writing a test around the command but honestly, I have no idea how set up the testing harness when working on a package (and bootstrapping Laravel etc., etc., etc.). So, thanks for your help. The only significant difference between our versions are the one line about the password. I changed it to this:

```php
$password = Hash::make(env('APP_KEY') . rand(1, 1000));
```

This way, users will use the standard password reset mechanism to log in and I won't be hard-coding passwords anywhere. Also, they can't pass the password in via the command line so it isn't logged in the command history.

Thanks for your help!
